chameleon/settings.py

```diff
@@ -250,9 +252,19 @@
     os.path.join(BASE_DIR, 'locale'),
 )
 
+## Keycloak OIDC Authentication
+OIDC_RP_CLIENT_ID = os.environ.get('OIDC_RP_CLIENT_ID')
+OIDC_RP_CLIENT_SECRET = os.environ.get('OIDC_RP_CLIENT_SECRET')
+OIDC_RP_SIGN_ALGO = os.environ.get('OIDC_RP_SIGN_ALGO')
+OIDC_OP_JWKS_ENDPOINT = os.environ.get('OIDC_OP_JWKS_ENDPOINT')
```

Comment:
Are all of these endpoints necessary? I'm not sure that we need the JWKS endpoint, for example.

Reply:
Ah yes, so the default doesn't work for Keycloak's default, makes sense. 👍